File: LeetCode_3289/Main.py
from typing import List

class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        answer = []
        cnt = [0 for _ in range(101)]

        for num in nums:
            cnt[num] += 1

            if cnt[num] == 2:
                answer.append(num)
        return answer

File: LeetCode_3289/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        int n = nums.size();
        vector<int> answer, cnt(n - 2, 0);

        for (int x : nums) {
            cnt[x] += 1;

            if (cnt[x] == 2) {
                answer.push_back(x);
            }
        }
        return answer;
    }
};